fern/api/generators.yml

```diff
@@ -1,4 +1,14 @@
 groups:
+  local:
+    generators:
+      - name: fernapi/fern-go-sdk
+        version: 0.3.0
+        config:
+          module:
+            path: github.com/hookdeck/hookdeck-go-sdk
```

Comment:
awesome, this was something that was blocking me from setting this up myself 😅  couldn't quite find any documentation for this on the Fern website

Reply:
It will eventually be included in Fern's main documentation, but it's included in the `fern-go` generator for now ([link](https://github.com/fern-api/fern-go#module)).